front/app/api/graph_data_units/responseTypes.ts

```diff
@@ -49,21 +49,26 @@ export type GroupedAnswer = Answer & {
   groups: { group: string | null; count: number }[];
 };
 
```

Comment:
The `Answer` type in this file, line 43-46, will be identical to the one in `api/survey_results/types`. Maybe we should reuse it?